### PR TITLE
Hook up model summary watcher to API

### PR DESCRIPTION
**Pull request by howbazaar**
Add WatchModelSummaries to client's Controller facade.

An admin oriented version WatchAllModelSummaries is also added.

The controller facade API version is bumped to 9 with the addition of these to methods.

The controller facade structure now needs the cache/controller in order to create the watcher types.

API schema is updated for the new version.

## QA steps

```sh
cd apiserver/facades/client/controller
go test
```

### CODE REVIEW

File: apiserver/facades/client/controller/controller.go [hunk 1]
```diff
@@ -196,10 +220,11 @@ func NewControllerAPI(
 		presence:            presence,
 		hub:                 hub,
 		multiwatcherFactory: factory,
+		controller:          controller,
 	}, nil
 }
 
-func (c *ControllerAPI) checkHasAdmin() error {
+func (c *ControllerAPI) checkIsSuperuser() error {
```
Comment (tlm) [inline apiserver/facades/client/controller/controller.go]: Should this be checkIsSuperUser ?

Comment (howbazaar) [reply apiserver/facades/client/controller/controller.go]: Yep, fixing.

**Comment by howbazaar** [comment]
$$merge$$

**Comment by howbazaar** [comment]
$$merge$$

**Pull request merged**